netlify.toml

```diff
@@ -10,6 +10,7 @@
   REACT_APP_BRIDGE_CREDIT_CLASS_ID = "C03"
   REACT_APP_GA_MEASUREMENT_ID="G-6ZGJM8JXYN"
   REACT_APP_TOUCAN_LINK="https://toucan.earth/overview?chainId=137"
+  REACT_APP_SKIPPED_CLASS_ID="C01"
```

Comment:
Should this be an array?
We might have more to add in the future.

Reply:
I was hesitating too...
what do you think @erikalogie, do you believe we might want to hide more credit classes in the future?

Reply:
@blushi I can ask Cory and Sarah but I don't think so

Reply:
yeah it seems like we won't: https://regen-network.slack.com/archives/C03392VF9N0/p1680535838110129?thread_ts=1680521682.518939&cid=C03392VF9N0
so fine to leave this as a simple string I guess